sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -1016,18 +1017,40 @@ class CompactBlockProcessor internal constructor(
     suspend fun getLastScannedHeight() =
         repository.lastScannedHeight()
 
+    // TODO(str4d): CompactBlockProcessor is the wrong place for this, but it's where all the other APIs that need
+    //  access to the RustBackend live. This should be refactored.
+    internal suspend fun createAccount(seed: ByteArray): UnifiedSpendingKey =
```

Comment:
This is related to the layered architecture we've always wanted. It's probably a good discussion to have in terms of kotlin multiplatform